qpc/cred/utils.py

```diff
@@ -45,6 +45,12 @@ def get_password(args, req_payload, add_none=True):
         req_payload['become_password'] = pass_prompt or None
     elif add_none:
         req_payload['become_password'] = None
+    if "token " in args and args.token:
```

Comment:
why this is space is expected ("token ")? and why args.token is not enough?
is this a protection against AttributeError? if so, `if getattr(args, "token", None)` would be enough

Reply:
the space in token was an accident. I will push a fixup with a correction.

Reply:
@bruno-fs IT does need to have both if "token " in args and args.token, otherwise it will raise an attributeError.

Reply:
ok. I suggest using getattr, then (see my first comment in this thread)